components/brave_vpn/browser/connection/brave_vpn_os_connection_api.h

```diff
@@ -133,6 +139,14 @@ class BraveVPNOSConnectionAPI
       mojom::ConnectionState::DISCONNECTED;
   BraveVPNRegionDataManager region_data_manager_;
   base::ObserverList<Observer> observers_;
+  // Used for tracking if the VPN dependencies have been installed.
+  // If the user has Brave VPN purchased and loaded with this profile
+  // AND they are a system user, we should call this once per session.
```

Comment:
1. Do they need to be a system user when we are using the elevation service?
2. The comment says "we should call this once per session". What's meant by "this"?

Reply:
One per browser-open is what I mean. Should get called again if person closes browser and re-opens. I'll change wording 😄

Reply:
Updated comment - see https://github.com/brave/brave-core/pull/20754/commits/2822bcd99aff26e6488ef96f9b73ec14f9771ad9